src/partials/50_section-products.html

```diff
@@ -253,10 +239,8 @@ <h5>Aenean Ru Bristique</h5>
           <div class="line"></div>
           <div class="actions">
             <div class="outlines">
-              <a href="#" class="btn-outline actives"><i class="far fa-heart"></i></a>
-              <a href="#" class="btn-outline actives"
-                ><i class="fas fa-exchange-alt"></i
-              ></a>
+              <a href="#" class="btn-outline active"><i class="far fa-heart"></i></a>
+              <a href="#" class="btn-outline"><i class="fas fa-exchange-alt"></i></a>
             </div>
```

Comment:
Dodałaś w każdym produkcie klasę 'active' do buttonu ulubione, a w żadnym produkcie do buttonu 'dodany do porównania' . Według polcenia :
_'W podglądzie projektu musi być co najmniej po jednym produkcie, który ma klasy active: 

    na żadnym z tych buttonów
    tylko na "ulubiony"
    tylko na "dodany do porównania"
    na obu buttonach_

Reply:
moze nie zrozumialam polecenia. Wydawwalo mi sie ze w poleceniu:
"na żadnym z tych buttonów
tylko na "ulubiony"
tylko na "dodany do porównania"
na obu buttonach_" moge sobie wybrac gdzie dodam klase .active

Reply:
chodzi jednak o to, aby każda z opcji znajdowała się przynajmniej w jednym produkcie. Popraw proszę ;)

Reply:
zad poprawione